benefits/static/css/styles.css

```diff
@@ -371,6 +374,10 @@ footer .footer-links li a:visited {
   line-height: 1;
 }
```

Comment:
I know this PR didn't introduce it, but I think we can fix a small bug in the Login.gov button logo.

We should add `background-repeat: no-repeat;` to this section of rules for the fallback text.

## Without this fix:

Here it is in `dev` right now in Chrome at tablet width showing the repeated image (appears like the shield is underlined):

![image](https://github.com/cal-itp/benefits/assets/1783439/8f5d01ad-c2d4-4ae9-bcc3-f6813b828964)

Here it is from this PR, the modal button, where it is much more obvious in color:

![image](https://github.com/cal-itp/benefits/assets/1783439/ebd20124-5f0b-4a12-a497-43b34bfe7afc)

## With the fix

Add one rule:

```css
#login .fallback-text {
  background-size: contain;
  background-repeat: no-repeat; /* <-- this one */
 /* ... */
}
```

Solves the issue on all buttons:

![image](https://github.com/cal-itp/benefits/assets/1783439/dfc4e71d-6fab-4adf-a446-38016527e56e)

Reply:
Ah, I think it was right here from yesterday, snuck by: https://github.com/cal-itp/benefits/pull/1508/files#diff-744df555aee38173a0b0868baa7befa720e6a28f80747523e74e724c126cd381

Reply:
Oh I see. Thanks for catching that!